proto/qvesting/query.proto

```diff
@@ -64,19 +102,16 @@ message QuerySpendableBalancesResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
-// QueryVestingAccountsRequest is the request type for the Query/Accounts RPC method.
-message QueryVestingAccountsRequest {
-  // pagination defines an optional pagination for the request.
-  cosmos.base.query.v1beta1.PageRequest pagination = 1;
+// QuerySpendableSupplyRequest is the request type for the Query/SpendableSupply RPC method.
+message QuerySpendableSupplyRequest {
```

Comment:
This query is good for the total floating amount? right
Total - locked in vesting or delegation.